lib/alchemy/eth_getLogs.tsx

```diff
@@ -1,4 +1,4 @@
-import { error } from "console"
+/* eslint-disable no-console */
```

Comment:
can you disable only the line `console.error` is being used instead of muting for the entire file?